### DISCUSSION TITLE
Restart network using geth nodes backup on different machine

### DISCUSSION THREAD

**Discussion by trinayanbhatt** [ConsenSys/quorum#1618] [Q&A]
We have mapped mounted docker volumes to store backup of the geth nodes. 

Now we are trying to restart those nodes on some other machine using the mounted docker volumes but somehow the nodes are exiting when we start the docker services for those geth nodes.

Can someone please help with this issue?

**Comment by baptiste-b-pegasys** [comment]
How do you make the backup?

**Comment by trinayanbhatt** [comment]
We have mapped the data directory of geth nodes with the external backup directory that we have created, and have mounted those as docker volume.

**Reply by antonydenyer** [reply]
You need to at the very least gracefully shutdown the geth node before doing anything with the volume. Or you could use some sort of backup and restore process.

https://geth.ethereum.org/docs/fundamentals/backup-restore
